lib/homepage.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:tutorial/constants.dart';
+
+class Homepage extends StatefulWidget {
+  const Homepage({super.key});
+
+  @override
+  State<Homepage> createState() => _HomepageState();
+}
+
+class _HomepageState extends State<Homepage> {
+  int currentPageIndex = 0;
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        // appBar: AppBar(
+        //   backgroundColor: primaryColor,
+        //   // title: const Text("Hello, Daniel!! :)"),
+        // ),
+        body: switch (currentPageIndex) {
+          1 => HomeScreen2(),
```

Comment:
these need to be renamed to the actual pages they are.  If we don't have them yet, or don't know the names, then add a TODO: here

Reply:
Change the numbers for which subpage we are showing to an enum